tap_kafka/sync.py

```diff
@@ -157,7 +179,7 @@ def read_kafka_topic(consumer, local_store, kafka_config, state, fn_get_args):
             break
 
         message = polled_message
-        LOGGER.info("%s:%s:%s: key=%s value=%s" % (message.topic(), message.partition(),
+        LOGGER.debug("%s:%s:%s: key=%s value=%s" % (message.topic(), message.partition(),
                                                    message.offset(), message.key(),
                                                    message.value()))
```

Comment:
Perhaps it's not good to log the message value too, for the sake of not exposing sensitive data.

Reply:
not even in debug mode? BTW, the most recent release of tap-kafa is debug logging the message. It was change accidentally by a previously merged commit and this line is reverting it to the original state.

Would you prefer removing it completely, or debug logging is fine but we should remove the actual message?

Reply:
we can leave the debug but don't include the message value I guess

Reply:
done by 6b37b1b72ed372cb4f0ca1b259ad82fde8749045